### ISSUE TITLE
Will this be able to export animations?

### ISSUE THREAD

**Issue by ecumber** [KillzXGaming/BFRES-Viewer#4]
Can you make this able to export as 3ds max animations?

**Comment by ecumber** [comment]
Also, how do you build this?

**Comment by KillzXGaming** [comment]
I don't really plan on working on this tool anymore as I have a better WIP tool I plan on releasing soon with animation exporting included. To build download visual studio and open the .sln file. Then build it

**Comment by ecumber** [comment]
are you working on the smash forge fork?

**Comment by KillzXGaming** [comment]
Yes although the latest commit is a bit old and it cannot rip anything. I plan on adding that in later when I fix all the bugs.
